docs/source/solving-interface.rst

```diff
@@ -734,12 +734,17 @@ system.  We describe the most commonly modified options (along with
 their defaults in Firedrake) here.  For linear variational solves we
 use:
 
-* ``ksp_type``: GMRES, with a restart (``ksp_gmres_restart``) of 30
-* ``ksp_rtol``: 1e-7
-* ``ksp_atol``: 1e-50
-* ``ksp_divtol`` 1e4
-* ``ksp_max_it``: 10000
-* ``pc_type``: ILU (Jacobi preconditioning for mixed problems)
+source/apt_deps.txt
+* a == L
+solver_parameters: {
+   * ``mat_type``: "aij"
+   * ``ksp_type``: "preonly"
+   * ``ksp_rtol``: 1e-7
+   * ``pc_type``: "lu"
+   * ``pc_factor``: _DEFAULT_DIRECT_SOLVER_PARAMETERS
+}
+
+`LinearVariationalSolver defaults <https://github.com/firedrakeproject/firedrake/blob/master/firedrake/variational_solver.py#L134-L143>`_
```

Comment:
This is just supposed to be a bulleted list. Please run `make -C docs html` and check the rendered HTML to make sure things look right.